devops/azure-pipelines.yml

```diff
@@ -0,0 +1,91 @@
+# This pipeline does not implement the analysis template from the 
+# project-registry-devops repository because there was an issue with
+# analysis of an Angular build when checking out multiple repos.
+
+# This pipeline will trigger on commits or pull-requests made to the branch-dev
+# or main branches.  Note that static analysis with SonarCloud is only performed
+# on "long-lived" branches, which are designated as 'main' and any branch name
+# prefixed with 'branch-'.
+trigger:
+- branch-dev
+- main
+pr:
+- branch-dev
+- main
+# The sonarcloud and discord variable groups are required for their respective
+# secrets to be available in the analysis template
+variables:
+- group: project-registry-sonarcloud
+- group: project-registry-discord
+- name: microservice
+  value: "frontend"
+- name: coveragePath
+  value: "./p3-frontend/**"
+- name: workDir
+  value: "./p3-frontend/"
+
+pool:
+  vmImage: ubuntu-latest
+
+stages:
+- stage: build
+  jobs:
+  - job: analysis
+    steps:
+      - checkout: self
+
+      - task: SonarCloudPrepare@1
+        inputs:
+          SonarCloud: 'SonarCloud'
+          organization: $(sonarOrg)
+          scannerMode: 'CLI'
+          configMode: 'manual'
+          cliProjectKey: $(sonarKeyPrefix)${{ variables.microservice }}
+          cliSources: './${{ variables.workDir }}'
+        displayName: Prepare Analysis for ${{ variables.microservice }} --> Task
+            
+      - task: Npm@1
+        inputs:
+          command: 'install'
+          workingDir: './${{ variables.workDir }}'
+        displayName: Install dependencies for ${{ variables.microservice }} --> Task
+
+      - task: AngularCLI@1
+        inputs:
+          command: 'build'
+          project: './${{ variables.workDir }}'
+          prod: false
+          DisableAnalytics: true
+        displayName: Build ${{ variables.microservice }} --> Task
+
+      - task: SonarCloudAnalyze@1
+        displayName: Analyze ${{ variables.microservice }} --> Task
+
+      - task: SonarCloudPublish@1
+        condition: always()
+        inputs:
+          pollingTimeoutSec: '300'
+        displayName: Publish Analysis of ${{ variables.microservice }} --> Task
+
+      - task: 'ado-discord-webhook@1'
+        condition: always()
+        inputs:
+          channelId: $(discordAnalysisChannel)
+          webhookKey: $(discordAnalysisKey)
+          name: ${{ variables.notifactionDisplayName }}
+          messageType: 'content'
+          content: | 
+                **__Analysis Report__**
```

Comment:
the indentation here is deeper than usual for no reason

Reply:
Fixed in new commit.